addons/survey/wizard/survey_invite.py

```diff
@@ -137,12 +137,16 @@ def _onchange_partner_ids(self):
                         _('The following recipients have no user account: %s. You should create user accounts for them or allow external signup in configuration.' %
                             (','.join(invalid_partners.mapped('name')))))
 
-    @api.onchange('template_id')
-    def _onchange_template_id(self):
-        """ UPDATE ME """
-        if self.template_id:
-            self.subject = self.template_id.subject
-            self.body = self.template_id.body_html
+    @api.depends('template_id')
+    def _compute_template_values(self):
+        for invite in self:
+            if invite.subject is None:
```

Comment:
Maybe something like
```
if not invite.subject: invite.subject = ''
if not invite.body: invite.body = ''
```
Really don't like those is None. Too much dependent on some low level implementation.